README.md

```diff
@@ -103,9 +103,11 @@ user able to run the docker commands.
 ## Logs
 
 Use the usual `docker ps` command to see which containers are running and the
-`docker logs {container_id}` to see logs for a given container.
+`docker logs {container_id}` to see logs for a given container. Also, logs may
+be found via `journalctl` with the `CONTAINER_NAME` as a filter, for example:
```

Comment:
This would be a good place to practice [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/)! That'd reduce the scope of the diff, so that there is not a removed line here.

Reply:
I like the concept but I also like the idea of a a readable-in-plaintext markdown. Not that it's not readable. I might try it in future, though.